WildShape-Sheets-API/Controllers/UserController.cs

```diff
@@ -38,5 +44,84 @@ public class UserController : Controller {
             userService.DeleteUser(id);
         }
 
+        public class PasswordResetRequest
```

Comment:
This seems like something that would possibly be better being made separately in model or DTO i think